frontend/css/cards.css

```
@@ -0,0 +1,435 @@
+    font-weight: 800;
+    color: var(--text-primary);
+}
+
+.match-vs {
+    display: none;
+}
+
+box-shadow: 0 1px 3px rgba(0, 0, 0, 0.3);
+border: 1px solid rgba(255, 255, 255, 0.15);
+flex-shrink: 0;
+}
```

Stylelint (17.14.0)

[error] Unexpected } (CssSyntaxError)

(CssSyntaxError)